quickstart/config/nginx/app-provider/swagger.conf

```diff
@@ -0,0 +1,10 @@
+server {
+    listen 80;
+    server_name swagger.localhost;
+    location /docs/openapi {
+        proxy_pass http://participant-app-provider:7575/docs/openapi;
+        add_header Access-Control-Allow-Origin *;
```

Comment:
Added headers here to allow CORS, avoiding the issue mentioned [here](https://github.com/swagger-api/swagger-ui/issues/8469)